packages/fe-fpm-writer/src/common/event-handler.ts

```diff
@@ -51,17 +52,18 @@ export function applyEventHandlerConfiguration(
     eventHandler: EventHandlerConfiguration | true | string,
     controllerSuffix = false,
     typescript?: boolean,
-    parameters: EventHandlerTypescriptParameters = defaultParameter
+    parameters: EventHandlerTypescriptParameters = defaultParameter,
+    defaultConfig = { templatePath: 'common/EventHandler', fnName: 'onPress', fileName: config.name }
```

Comment:
Should this be a part of the `config` parameter already defined?

Reply:
I removed the fileName property since it is available in config.